Support/Scripts/run-lldb-tests.sh

```diff
@@ -55,6 +60,7 @@ opt_no_upstream_blacklists=false
 opt_log=false
 opt_strace=false
 opt_use_lldb_server=false
+DOSTEP_EXTRA_ARGS=""
```

Comment:
You should probably name this opt_dostep_... to match with the other option variables.

Reply:
It's not an option, though. `arg_dostep_extra_args`?

Reply:
It's not true/false but it's a command-line option. No?